metrics/New_Contributor_Closing_Issues.md

```diff
@@ -1,27 +1,28 @@
 # New Contributors Closing Issues
 
-Question: How many contributors are closing issues for the first time?
+Question: How many contributors are closing issues for the first time in a given project?
 
 ## Description
-This metric is an indication of the volume of contributors who are closing issues for their first time. When a contributor closes an issue for the first time it is some indication of "stickiness" for the individual within a project, especially for contributors who are not also committers.
+This metric is an indication of the volume of contributors who are closing issues for their first time within a given project. When a contributor closes an issue for the first time it is some indication of "stickiness" of the individual within that project, especially for contributors who are not also committers.
```

Comment:
See comment above ("project" -> "repository")

Reply:
Same remark as above. I will let you choose whatever you feel more appropriate.